js/widgets/widgetWindows.js

```diff
@@ -129,18 +129,7 @@ class WidgetWindow {
         this._nonclose.onmousedown = (e) => {
             this._dragging = true;
```

Comment:
couldn't this simply be this._draging = !this_maximized;

Reply:
i have updated it plz check